src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -119,6 +119,13 @@ class BookingFacade(
     sendBookingEmails(BookingAction.COURT_HEARING_LINK_REMINDER, videoBooking, getPrisoner(videoBooking.prisoner()), user)
   }
 
+  fun sendProbationOfficerDetailsReminder(videoBooking: VideoBooking, user: ServiceUser) {
+    require(videoBooking.isBookingType(PROBATION)) { "Video booking with id ${videoBooking.videoBookingId} is not a probation booking" }
+    require(videoBooking.probationTeam!!.enabled) { "Video booking with id ${videoBooking.videoBookingId} is not with an enabled probation team" }
+    require(videoBooking.appointments().any { it.appointmentDate > LocalDate.now() }) { "Video booking with id ${videoBooking.videoBookingId} has already taken place" }
+    sendProbationBookingEmails(BookingAction.PROBATION_OFFICER_DETAILS_REMINDER, videoBooking, getPrisoner(videoBooking.prisoner()), user)
```

Comment:
Should the date require() include today, or only from tomorrow onwards?

Reply:
I can add in extra validation.  There is a second PR to come.